testing-framework/ec2/base/bootstrap_node_evm_provisioner.tf

```diff
@@ -0,0 +1,164 @@
+locals {
+  bootstrap_nodes_evm_ip_v4 = flatten([
+    [aws_instance.bootstrap_node_evm.*.public_ip]
+    ]
+  )
+}
+
+resource "null_resource" "setup-bootstrap-nodes-evm" {
```

Comment:
hmmm I thought we already have bootnodes for domains 🤔

Reply:
The ephemeral devnet requirements changed, so i added a bootnode for domains.